test/e2e_node/oomkiller_linux_test.go

```diff
@@ -92,19 +92,21 @@ var _ = SIGDescribe("OOMKiller [LinuxOnly]", framework.WithNodeConformance(), fu
 		})
 	}
 	for _, tc := range testCases {
-		runOomKillerTest(f, tc, 0)
+		runOomKillerTest(f, tc, 0, "UnlimitedSwap")
```

Comment:
UnlimitedSwsp does not exist anymore. The other option would be NoSwap.

Reply:
Thank you for your comment. I'll fix.